lib/lighthouse/benefits_documents/configuration.rb

```diff
@@ -0,0 +1,133 @@
+# frozen_string_literal: true
+
+require 'common/client/configuration/rest'
+require 'lighthouse/auth/client_credentials/jwt_generator'
+require 'lighthouse/auth/client_credentials/service'
+
+module BenefitsDocuments
+  ##
+  # HTTP client configuration for the {BenefitsClaims::Service},
+  # sets the base path, the base request headers, and a service name for breakers and metrics.
+  #
+  class Configuration < Common::Client::Configuration::REST
+    self.read_timeout = Settings.lighthouse.benefits_documents.timeout || 20
+
+    SYSTEM_NAME = 'vets.gov'
```

Comment:
For Lighthouse, I don't think you _need_ a system name. But if they require one, or you just want to include it, let's change it to the more modern `va.gov` if we can.

Reply:
+1

Reply:
I left the name in because a couple other lighthouse services still include the system name. I updated it to `va.gov` though